v20.1/time.md

```diff
@@ -63,7 +63,19 @@ A `TIME` column supports values up to 8 bytes in width, but the total storage si
 
 A `TIMETZ` column supports values up to 12 bytes in width, but the total storage size is likely to be larger due to CockroachDB metadata.
 
-## Example
+## Precision
+
+<span class="version-tag">New in v20.1:</span> CockroachDB supports precisions from 0 (seconds) to 6 (microseconds) for `TIME`/`TIMETZ` values. By default, `TIME`/`TIMETZ` values have a precision of 6 (microseconds).
```

Comment:
Should we define what `precision` means or is that common knowledge? 
Also, is `precisions` the correct term? (It might be; I am unsure about it because I don't understand what precision means).
The example in the following section doesn't help me understand how/why/when to use it either.

(Sorry if all of this is common knowledge, in which case, please excuse my ignorance)